cygwin.rst

```diff
@@ -0,0 +1,194 @@
+*****************************
+Docs in Windows (Cygwin) 
+*****************************
+
+.. _cygwin-docs:
+
+Cygwin
+-------------------------------
+Cygwin is windows implementation of unix-like commands. You can basically use many linux commands from windows 
+command propmpt.
+
+Install `cygwin <https://cygwin.com/install.html/>`_ and add its path into windows environmental variable.
+This will allow you to `use cygwin from cmd promopt <https://www.howtogeek.com/howto/41382/how-to-use-linux-commands-in-windows-with-cygwin/>`_.
+
+
+.. _python:
+
+Python 3
+-------------------------------
+You need to install `python 3 <https://www.python.org/downloads/>`_.
+During the installation, you can select to set up pip tool as well as add python path to the Path environment variable.
```

Comment:
Provide instructions or link to instructions.

Reply:
Is it possible to add video for python installation?

Reply:
Are you saying you want to make a video to include? Or do you want to link to an existing video?

Reply:
A link for an existing one

Reply:
Seems reasonable.